src/Components/ButtonPanel.js

```diff
@@ -11,7 +11,7 @@ const buttonGroups = {
 };
 
 const ButtonPanel = (props) => (
```

Comment:
- [ ] From the instructions from pathwright, `handleClick` is implemented as a function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png))
- [ ] Then  `handleClick` is passed as `clickHandler` props to each `Button` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png))